dev/io.openliberty.transport.ssl_fat/bnd.bnd

```diff
@@ -0,0 +1,53 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
```

Comment:
If you can remove the contributors from these classes / files that would be a good clean up